connector_redmine/README.rst

```diff
@@ -0,0 +1,67 @@
+Redmine Connector
```

Comment:
@dufresnedavid  Thanks for your contribution to the OCA. 
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

Can you please use latest README template?